driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
Ah I see,  I added a `@BeforeClass` annotation and it got invoked after `CCMBridge` kicked in, but I see how overriding this could do the trick.  I'll do that.